### PR TITLE
🚸 Fix connection timeout

### PR DESCRIPTION
**Pull request by falexwolf**

### CODE REVIEW

**Comment by Koncopd** [comment]
One downside of this is that it degrades the performance for local postgres, sqlite etc.

File: lamindb_setup/dev/django.py [hunk 1]
```diff
@@ -73,6 +75,12 @@ def get_migrations_to_sync():
     return status, latest_migrs
 
 
+def close_if_health_check_failed(self) -> None:
+    if self.close_at is not None and time.monotonic() >= self.close_at:
+        self.close()
+    self.close_at = time.monotonic() + CONN_MAX_AGE
```
Comment (Koncopd) [inline lamindb_setup/dev/django.py:81]: this should be only reset if it is not `None` i think.

Comment (Koncopd) [reply lamindb_setup/dev/django.py:81]:
```
if self.close_at is not None:
  self.close_at = time.monotonic() + CONN_MAX_AGE
```

Comment (falexwolf) [reply lamindb_setup/dev/django.py]: Agreed! Fixed below!

**Pull request merged**